examples/Benchmark.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Oct 18 15:53:27 2016
+
+@author: Michael Wu
+
+Benchmark test
+Giving performances of RF(scikit) and MultitaskDNN(TF)
+on datasets: muv, nci, pcba, tox21, toxcast, sider
+
+time estimation(on a nvidia tesla K20 GPU):
+tox21 - dataloading: 30s
+      - tf: 40s
+muv   - dataloading: 400s
+      - tf: 250s
+pcba  - dataloading: 30min
+      - tf: 2h
+sider - dataloading: 10s
+      - tf: 60s
+toxcast dataloading: 70s
+	tf: 70s
+(will include more)
+"""
+from __future__ import print_function
+from __future__ import division
+from __future__ import unicode_literals
+
+import sys
+import os
+import numpy as np
+import shutil
+import time
+import deepchem as dc
+
+from sklearn.linear_model import LogisticRegression
+from sklearn.ensemble import RandomForestClassifier
+
+from muv.muv_datasets import load_muv
+from nci.nci_datasets import load_nci
+from pcba.pcba_datasets import load_pcba
+from tox21.tox21_datasets import load_tox21
+from toxcast.toxcast_datasets import load_toxcast
+from sider.sider_datasets import load_sider
+
+def benchmark_loading_datasets(base_dir_o, hyper_parameters, n_features = 1024, 
+                               dataset_name='all',model='all',reload = True,
+                               verbosity='high',out_path='/tmp'):
+  """
+  Loading dataset for benchmark test
+  
+  Parameters
+  ----------
+  base_dir_o, string
```

Comment:
The format of this docstring is a little off (see https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt). For example, need
`base_dir_o: str` instead. Could you change the docstrings in this file?